### DISCUSSION TITLE
[Logstash.outputs.elasticsearch] Strelka indexing error

### DISCUSSION THREAD

**Discussion by samanosuke26** [Security-Onion-Solutions/securityonion#11463] [General]
SO version 2.3.130 (However I'm running the so-strelka-backend 2.3.190 image.)

I'm receiving errors indexing strelka logs.
"status"=>400, "error"=>{"type"=>"illegal_argument_exception", "reason"=>mapper [scan.pe.subsystem_version\ cannot be changed from type [long] to [float]"}

Any assistance with this would be great, thank you.

**Comment by dougburks** [comment]
> SO version 2.3.130 (However I'm running the so-strelka-backend 2.3.190 image.)

We don't support running different versions of docker images like this. [2.3.130 is over a year old](https://blog.securityonion.net/2022/06/security-onion-23130-now-available.html) so we'd recommend upgrading to [2.3.260](https://blog.securityonion.net/2023/06/security-onion-23260-now-available.html) OR migrating to [2.4](https://blog.securityonion.net/2023/08/security-onion-24-has-reached-general.html).
